primitives/rpc/debug/src/api/single.rs

```diff
@@ -101,26 +102,26 @@ pub struct RawStepLog {
 
 #[derive(Clone, Eq, PartialEq, Debug, Encode, Decode)]
 #[cfg_attr(feature = "std", derive(Serialize))]
-#[cfg_attr(feature = "std", serde(rename_all = "camelCase", tag = "type"))]
+#[cfg_attr(feature = "std", serde(rename_all = "lowercase", tag = "type"))]
 pub enum CallInner {
 	#[cfg_attr(feature = "std", serde(rename_all = "camelCase"))]
```

Comment:
It is not readable to mix lowercase and camelCase. It is better to use neither of them and to rename specifically the only 2 fields to rename (call_type and SelfDestruct).

Reply:
I will sneak this in https://github.com/PureStake/moonbeam/pull/847